### PR TITLE
Add devTools.updateStore

### PR DESCRIPTION
**Pull request by jhen0409**
Related to #35.

We should have a same API with `window.devToolsExtension.updateStore` of Redux DevTools Extension.

### CODE REVIEW

File: src/devTools.js [hunk 1]
```diff
@@ -303,3 +303,10 @@ export default function devTools(options = {}) {
     };
   };
 }
+
+devTools.updateStore = (newStore) => {
+  // Mutate the store in order to keep the reference
```
Comment (zalmoxisus) [inline src/devTools.js]: Why not just `store = newStore`?

Comment (jhen0409) [inline src/devTools.js]: You're right, it's done. 😂

Comment (zalmoxisus) [inline src/devTools.js]: Thanks! I wasn't sure about that :)

**Comment by zalmoxisus** [comment]
Thanks a lot, @jhen0409! Unfortunately, there's no other solution for now than updating or adding the second enhancer, which should be the first in the compose. Thought the latter is even more confusing.

**Comment by zalmoxisus** [comment]
`devTools.updateStore` is deprecated, as `composeWithDevTools` does the trick. More details in [the docs](https://github.com/zalmoxisus/remote-redux-devtools#use-devtools-compose-helper).

**Pull request merged**